orgs/altinn-orgs.json

```
@@ -413,6 +413,16 @@
                 "production"
             ]
         },
+        "ks": {
```

Biome (2.4.15)

[error] The key ks was already declared.

(lint/suspicious/noDuplicateObjectKeys)